pkg/asset/ignition/machine/node.go

```diff
@@ -21,7 +21,7 @@ func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, ro
 					Source: func() *url.URL {
 						return &url.URL{
 							Scheme: "https",
-							Host:   fmt.Sprintf("api-int.%s:22623", installConfig.ClusterDomain()),
+							Host:   fmt.Sprintf("%s:22623", installConfig.BareMetal.ApiVIP),
```

Comment:
Isn't this changing generic code, and not baremetal platform specific?

Reply:
Yes. The current version of this is based on Roy's work in https://github.com/rgolangh/installer/commit/69f973204473e567aa599066228b7751df745c91 which changed common code. I need to look at how to make it upstreamable.

Reply:
Can't we just put some ifs?

Reply:
Yeah, that's what I ended up doing.